pages/common/ooniprobe.md

```diff
@@ -0,0 +1,33 @@
+# ooniprobe
+
+> Open Observatory of Network Interference.
+> Test the blocking of websites and apps. Measure the speed and performance of your network.
+> More information: <https://ooni.org/>.
```

Comment:
```suggestion
> More information: <https://ooni.org/support/ooni-probe-cli/>.
```
I would suggest linking to the man page/docs directly here.

Reply:
Thanks for the suggestions. I should have implemented them all now.